ddtrace/internal/writer.py

```diff
@@ -155,6 +149,16 @@ def run_periodic(self):
             if not self._send_stats:
                 return
 
+            # Statistics about the writer thread
+            if hasattr(time, 'thread_time'):
+                new_thread_time = time.thread_time()
+                diff = new_thread_time - self._last_thread_time
+                self._last_thread_time = new_thread_time
+                self.dogstatsd.histogram('datadog.tracer.writer.cpu_time', diff)
+
+            # Report global stats
+            stats.get_stats().report(self.dogstatsd)
```

Comment:
I don't love this API, I think I'll change to be:

```python
# stats.py

def report(*args, **kwargs):
    global _stats
    return _stats.report(*args, **kwargs)

def patch_error(*args, **kwargs):
    global _stats
    return _stats.patch_error(*args, **kwargs)
```

Calling `get_stats` is cumbersome.